umm-spec-lib/src/cmr/umm_spec/spatial_conversion.clj

```diff
@@ -20,11 +21,15 @@
    "WRS-1"
    "WRS-2"])
 
+(def upcase-valid-tile-identification-system-names
+  "Upcase values to allow for case-insenitive validation"
+  (map util/safe-uppercase valid-tile-identification-system-names))
+
```

Comment:
My understanding about the case insensitive part is the following:
1. the valid-title-identification-system-names stay the same - It should still contain the lower/upper case mixed.
   This is the what the enum list allows. 
2. when converting from other format to UMM v1.10, and converting from UMM v1.9 to v1.10, 
     take the value "MODIS Tile EASE", as example,  you consider other "modis tILE ease" as valid values, which is 
     correct, but you still need to translate "modis tiLE ease" to "MODIS Tile EASE", which you didn't do. So, you need to 
     change the translate-tile-id-system-name function. Same with the vertical spatial domain.
3. Change the add unit tests to reflect these changes.